### DISCUSSION TITLE
Question for efficient data preparation for wflow.jl input!!

### DISCUSSION THREAD

**Discussion by DavidChoi76** [Deltares/Wflow.jl#495] [Q&A] [answered]
There is a description to prepare wflow.jl input in [here](https://deltares.github.io/Wflow.jl/dev/user_guide/model-setup/)
However, I don't think there's enough explanation to fully understand this process.

I don't know what the efficient approach is for data preparation to create wflow.jl input.
The only way I can think of is using netCDF4, numpy, and rasterio python library.
I will arrange each variable such as csv file format for forcing data and netcdf and raster for static data.
Then I will add each variable and data into netcdf files.

Can anyone recommend a better approach for this process?

**Comment by verseve** [comment, marked as answer]
Most efficient approach is to use the Python package [hydromt_wflow](https://github.com/Deltares/hydromt_wflow). Also mentioned in the docs [here](https://deltares.github.io/Wflow.jl/dev/user_guide/model-setup/#hydroMT).
